### ISSUE TITLE
Rule: S3 Bucket Default Encryption Enabled is not triggered by configuration change

### ISSUE THREAD

**Issue by svshane** [awslabs/aws-config-rules#99]
Changing the default encryption( None -> AES-256 or AES-256 -> None) in the S3 bucket does not trigger the rule re-evaluation, although it is stated in the template that it's change triggered:

`# Trigger Type: Change Triggered`

Re-evaluating via button inside the rule details after a change presents the correct evaluation results, so I assume my rule (and lambda) is working correctly.

**Comment by jongogogo** [comment]
Good catch, this rule is not correct. It should be a periodic rule, since encryption is not captured as part of the configuration item, hence not triggering the rule.

Would you like to give it a shot at refactor the rule into a periodic using the RDK?

**Comment by svshane** [comment]
Unfortunately I'm not a python developer. The reason I used this python rule was to see if the update of the encryption is triggering the rule. (which is now confirmed that it won't)

Meanwhile I implemented a periodic rule which checks the encryption status of all buckets but instead of python I used typescript compiled to JS (Node 8.10).

I could provide my code here, but TS is not listed so far and the compiled JS is not very pretty.

**Comment by jongogogo** [comment]
Noted, thanks. We keep the issue open, we will address it.
